scene/2d/audio_stream_player_2d.cpp

```diff
@@ -430,6 +461,39 @@ void AudioStreamPlayer2D::_on_bus_renamed(int p_bus_index, const StringName &p_o
 	notify_property_list_changed();
 }
 
+bool AudioStreamPlayer2D::_set(const StringName &p_name, const Variant &p_value) {
+	if (playback_parameters.has(p_name)) {
+		ParameterData &pd = playback_parameters[p_name];
+		pd.value = p_value;
```

Comment:
I think needs to detect if the variant is null, to handle the above issue

Reply:
Variant is null depends on the parameter, not all stream parameters will be necessarily exposed as checkable.

Reply:
Okay I figured it out, should not save it now.